config/config.exs

```diff
@@ -5,6 +5,7 @@ import Config
 
 config :edgehog_device_forwarder, EdgehogDeviceForwarderWeb.Endpoint,
   url: [host: "localhost"],
+  server: true,
```

Comment:
This is set only in prod for a reason (e.g. you don't want to start the server if you're just running migrations or some other `mix` task)
https://github.com/phoenixframework/phoenix/issues/3205
I don't know how to achieve this using `nix` but ideally you want to explicitly run `mix phx.server` instead, which starts the server